index.php

```diff
@@ -5,14 +5,46 @@
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
 if(isset($_POST['name'])) {
+    if(empty($_POST['name'])) {
+        echo 'A string is required';
+    } else {
+        $name = $_POST['name'];
+    }
+    if(empty($_POST['play_time_in_mins'])) {
+        echo 'A Number is required';
+    } else {
+        $playTime = $_POST['play_time_in_mins'];
+        if (preg_match('/[^0-9\s]+$/', $playTime)) {
+            echo 'Must be a number';
```

Comment:
this echos in kind of a random place, you might want to format it better so the user is more likely to notice it